testsuite/integration-arquillian/tests/other/base-ui/src/test/java/org/keycloak/testsuite/ui/account2/LinkedAccountsTest.java

```diff
@@ -171,6 +192,36 @@ public void unlinkAccountTest() {
         assertEquals(0, testUserResource().getFederatedIdentity().size());
     }
 
+    @Test
+    public void unlinkWebAuthnTest() {
+        signingInPage.navigateTo();
```

Comment:
Should we also test that unlinking the only IdP is not possible when there's no password or webauthn passwordless set?